src/features/landing/components/ForWhoSection/ForWhoSectionCardItem/ForWhoSectionCardItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { Card, Grid, Paper, Typography } from '@mui/material';
+import Image from 'next/image';
+import { useTranslation } from 'react-i18next';
+
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import { TRANSLATION_NAMESPACE } from '../../../utils/constants/constants';
+
+interface IForWhoSectionCardItemProps {
+  imageSrc: string;
+  imageAltText: string;
+  text: string;
+}
+
+const style = {
+  text: {
+    color: '#1A1C1E',
+    fontFamily: 'GolosText-Regular, sans-serif',
+    fontSize: '15px',
+    fontStyle: 'normal',
+    fontWeight: '400',
+    lineHeight: '25px',
+  },
+  card: {
+    padding: '27px 32px 28px 32px',
+    display: 'flex',
+    alignItems: 'center',
+    gap: '12px',
+    minHeight: '115px',
+    boxShadow: '0px 8px 27px 0px rgba(49, 59, 67, 0.14)',
+    borderRadius: '12px',
+  },
+};
+
+export default function ForWhoSectionCardItem({
```

Comment:
will be easiest to create just CardItem component that will be like a card item, and you can reuse this component easily in all application